data/json/items/comestibles/other.json

```diff
@@ -304,6 +304,64 @@
     "fun": 1,
     "vitamins": [ [ "calcium", 8 ], [ "vitB", 32 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "tofu",
+    "name": "tofu",
+    "name_plural": "tofus",
```

Comment:
No need to specify "name_plural" if it just has "s" added to the end, as it's the default way for game to form plural names. Same for below items.